templates/kubernetes-cluster-with-new-vpc.template

```diff
@@ -503,8 +503,11 @@ Outputs:
     Description: Run locally - SSH command to proxy to the master instance
       through the bastion host, to access port 8080 (command to SSH to the master Kubernetes node).
     Value: !Sub >-
-      ssh -A -L8080:localhost:8080
-      -o ProxyCommand='ssh ubuntu@${BastionHost.PublicIp} nc ${K8sStack.Outputs.MasterPrivateIp} 22'
+      SSH_KEY="path/to/${KeyName}.pem";
+      ssh
+      -i $SSH_KEY
+      -A -L8080:localhost:8080
+      -o ProxyCommand='ssh -i \"$\{SSH_KEY}\" ubuntu@${BastionHost.PublicIp} nc %h %p'
```

Comment:
...whereas the expansion of SSH_KEY is escaped (`$\{}`) so it will render as an unexpanded `${SSH_KEY}` when the user sees the Output.